README.md

```diff
@@ -107,8 +109,15 @@ On Linux (Ubuntu 17.10 LTS or later), install these with:
     apt-get -y install python3.6
     apt-get -y install python-pip
     apt-get -y install python-dev
+    apt-get -y install uuid-dev
+
+On other POSIX-like operating systems, `libuuid` and its header files
+may be in a package named `libuuid-devel` or `util-linux-libs`. The
```

Comment:
You also mentioned `uuid-dev` below, should it be mentioned here too?

Reply:
Oh, nevermind, I see the `apt-get -y install uuid-dev` above now.

Reply:
For whatever reason the current docs focus on ubuntu-family distros. I searched for the package names for the major families and these three are what I came up with (`util-linux-libs` is the post-May-2020ish name on Arch).